R/ppc-intervals.R

```diff
@@ -283,16 +298,18 @@ label_x <- function(x) {
   }
 
   grouped_d <- dplyr::group_by(molten_reps, !!! group_vars)
-  alpha <- (1 - prob) / 2
+  alpha <- (1 - probs) / 2
   probs <- sort(c(alpha, 0.5, 1 - alpha))
 
   val_col <- sym("value")
   dplyr::ungroup(dplyr::summarise(
     grouped_d,
     prob = prob,
-    lo = quantile(!! val_col, prob = probs[1]),
-    mid = quantile(!! val_col, prob = probs[2]),
-    hi = quantile(!! val_col, prob = probs[3])
+    ll = quantile(!! val_col, prob = probs[1]),
+    lo = quantile(!! val_col, prob = probs[2]),
+    mid = quantile(!! val_col, prob = probs[3]),
+    hi = quantile(!! val_col, prob = probs[4]),
+    hh = quantile(!! val_col, prob = probs[5])
   ))
```

Comment:
I agree. I like to have the outer and inner width as columns because that's good information to have.  Then you can combine multiple sets of intervals together.

```
library(bayesplot)
library(dplyr)
x <- example_mcmc_draws()
a1 <- mcmc_intervals_data(x, prob = .9)
a2 <- mcmc_intervals_data(x, prob = .7)
a3 <- mcmc_intervals_data(x, prob = .5)
a4 <- mcmc_intervals_data(x, prob = .3)
a5 <- mcmc_intervals_data(x, prob = .1)
all <- bind_rows(a1, a2, a3, a4, a5)

ggplot(all) + 
  aes(y = parameter, yend = parameter, size = inner_width) + 
  geom_segment(aes(x = l, xend = h)) + 
  scale_size(range = c(6, 1))
```

![image](https://user-images.githubusercontent.com/1890315/40857493-f30adcc6-65a0-11e8-9f20-64a97b7bf2eb.png)